### PR TITLE
Fix: use Escoria plugin version from plugin.cfg in savegames

### PR DESCRIPTION
**Pull request by StraToN**
Save manager used escoria.ESCORIA_VERSION constant value, which is now incorrect. We now use plugin.cfg's version instead.

### CODE REVIEW

**Comment by balloonpopper** [comment]
I can confirm this fixes the bug with the infinite loop, however I did find a edge case that isn't handled. I manually deleted save games 1-3 but still had save game 4 in the folder.
This results in an error on line 38 of load_game.gd
Invalid get index '1' (on base: 'Dictionary').

**Comment by StraToN** [comment]
Good point. I'll add a check to manage this case.

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,5 @@
 bin
 *.import
 .import
-*.translation
-.godot
\ No newline at end of file
+game/translations/*.translation
```
Comment (dploeger) [inline .gitignore:4]: What's the point of this in the context of this PR? If it doesn't make sense, please remove the change.

**Review by dploeger — CHANGES_REQUESTED** [review]
Just a very small remark.

**Review by BHSDuncan — APPROVED** [review]
I'm good with things and agree w Dennis.

File: .gitignore (hunk 1 above)
Comment (StraToN) [reply .gitignore]: Done

**Review by BHSDuncan — APPROVED** [review]

**Pull request merged**